lib/engine/game/g_18_ts/game.rb

```diff
@@ -0,0 +1,1166 @@
+# frozen_string_literal: true
+
+require_relative 'meta'
+require_relative '../base'
+require_relative '../../option_error'
+require_relative '../../distance_graph'
+require_relative 'entities'
+require_relative 'map'
+
+module Engine
+  module Game
+    module G18TS
+      class Game < Game::Base
```

Comment:
As this is an 1825 derivative which looks like it duplicates a lot of code from the 1825 implementation, it would be better if this was a subclass of `G1825::Game` instead of `Game::Base`. That would give you most of 1825’s rules without all this copied code. Though you might have to refactor the 1825 code in places to allow you to override behaviour here.